traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -98,8 +98,13 @@ extension AvailabilityViewController: UITableViewDataSource {
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if indexPath.row == 0 {
             let cell = tableView.dequeueReusableCell(withIdentifier: dateCellIdentifier, for: indexPath) as! DateCell
-            cell.valueLabel.textColor = (availabilityError != nil ? true : false) ? UIColor.red : UIColor.darkText
-            cell.titleLabel.textColor = (availabilityError != nil ? true : false) ? UIColor.red : UIColor.darkText
+            if #available(iOS 13.0, *) {
+                cell.valueLabel.textColor = (availabilityError != nil ? true : false) ? UIColor.red : UIColor.label
```

Comment:
You just need availabilityError != nil, no need for the ? true : false.  Also, you just make a var for the default label color based on if iOS13 or not rather than repeating the cell.valueLabel.textColor and cell.titleLabel.textColor assignments twice.